src/admin/python/lsst/qserv/admin/cli/entrypoint.py

```diff
@@ -441,6 +442,7 @@ def proxy(ctx: click.Context, **kwargs: Any) -> None:
         czar_cfg_file=targs["czar_cfg_file"],
         czar_cfg_path=targs["czar_cfg_path"],
         cmd=targs["cmd"],
+        log_cfg_file=targs["log_cfg_file"]
```

Comment:
(Perhaps) missing `,` in the end of the line?

Reply:
Hmm I'm surprised `black` didn't flag that. It's not strictly necessary but yes, it indicates to black that I want one item per line. And it's how I normally write it so it is better for consistency to add it.